### ISSUE TITLE
Can't even log in

### ISSUE THREAD

**Issue by bedantadey17** [hrkfdn/ncspot#1488]
**Describe the bug**
I can't log in with my spotify username, I am guessing the username goes something like 31eriwj............ and i am entering the password but it shows bad credentials

**To Reproduce**
1. Login

**Expected behavior**
Should be able to login

**Screenshots**
![image](https://github.com/user-attachments/assets/3b116b0c-4f7e-4cc6-8db8-64e4016f8c93)
![image](https://github.com/user-attachments/assets/4138821a-e004-4e60-b940-23b723224161)

**System (please complete the following information):**
 - OS: Linux
 - Terminal: GNOME Terminal
 - Version: Latest
 - Installed from: AUR

**Backtrace/Debug log**
NA

Instructions on how to capture debug logs: https://github.com/hrkfdn/ncspot#debugging

For backtraces, make sure you run a debug build of ncspot, e.g. by running the
command mentioned in the [compilation
instructions](https://github.com/hrkfdn/ncspot#compiling).  You can find the
latest backtrace at `~/.cache/ncspot/backtrace.log`.

**Additional context**
I have double, triple checked my credentials

**Comment by Callumari08** [comment]
Having this issue too. According to [this](https://forums.slimdevices.com/forum/user-forums/3rd-party-software/1721089-important-spotty-will-stop-supporting-credentials) forum post, spotify notified librespot that username-password auth will no longer work, instead you'll have to pass a token to login... I don't know how a lot of other people don't have this issue, though.

**Comment by lukebarone** [comment]
I got this working by rebuilding the app, and following the instructions at #1500

**Comment by hrkfdn** [comment]
Duplicate of #1500, this should be solved once we can draft a new release, however we're currently waiting on the release of the 3rd party dependency (librespot).
